Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -967,6 +1057,87 @@ def get_issues_command(client: Client, args: Dict[str, Any]) -> CommandResults:
     )
 
 
+def get_services_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    total_results, max_page_size = calculate_limits(args.get('limit', None))
+
+    provider = ','.join(argToList(args.get('provider')))
```

Comment:
You're right, in this case it seems a bit unnecessary. 

I think I pulled this from the existing `get_issues_command`, I'm guessing the original author of that did it for any additional protections that come from `argToList`. I'm tempted to leave it as it just in case the input isn't exactly as expected.